sql/src/main/java/io/crate/statistics/Stats.java

```diff
@@ -85,6 +85,13 @@ public long sizeInBytes() {
         return sizeInBytes;
     }
 
+    public long averageSizePerRowInBytes() {
+        if (numDocs == -1) {
+            return numDocs;
```

Comment:
Maybe return `-1` directly. 
Had a "wait what?" moment here because looking at `return numDocs` isolated I didn't get why it's okay to return that as `averageSizePerRowInBytes` value.